pam.c

```diff
@@ -100,6 +101,7 @@ void run_pw_backend_child(void) {
 		} else if (size == 0) {
 			break;
 		}
+		assert(pw_buf[size - 1] == '\0');
```

Comment:
We have a copy of the same path in `shadow.c` - maybe we should put the assert in `read_comm_request` itself?

Reply:
Good point!